spec/models/user_spec.rb

```diff
@@ -0,0 +1,57 @@
+require "rails_helper"
+require "./app/models/user"
+
+describe User do
+  context "when all attributes are correct" do
+    it "is created" do
+      user = User.create(name: "John Doe",
+                         email: "user@gmail.com")
+
+      expect(user).to be_a User
+    end
+  end
+
+  context "when name not given" do
+    it "raises an error" do
+      user = User.create(email: "user@gmail.com")
+
+      expect { user.save! }
```

Comment:
Why `.create` and then `.save!`

Once again, I would build first, call `user.valid?` and then check the error message.
But also, I would save me time and use shoulda-matchers

Reply:
Thanks Elle, again, ignore in next PRs as I will fix this across all specs